code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityMapTests.java

```diff
@@ -293,28 +270,28 @@ public void test_remove_nonexistentNamespaceAndItems() {
 		assertEquals(3, castedMap.get("login").size());
 	}
 
-
 	@Test
 	public void test_FromData() throws Exception {
 		// setup
-		final String jsonStr = "{\n" +
-							   "      \"identityMap\": {\n" +
-							   "        \"ECID\": [\n" +
-							   "          {\n" +
-							   "            \"id\":" + "randomECID" + ",\n" +
-							   "            \"authenticatedState\": \"ambiguous\",\n" +
-							   "            \"primary\": true\n" +
-							   "          }\n" +
-							   "        ],\n" +
-							   "        \"USERID\": [\n" +
-							   "          {\n" +
-							   "            \"id\":" + "someUserID" + ",\n" +
-							   "            \"authenticatedState\": \"authenticated\",\n" +
-							   "            \"primary\": false\n" +
-							   "          }\n" +
-							   "        ]\n" +
-							   "      }\n" +
-							   "}";
+		final String jsonStr =
+			"{\n" +
+			"      \"identityMap\": {\n" +
+			"        \"ECID\": [\n" +
+			"          {\n" +
+			"            \"id\":randomECID,\n" +
+			"            \"authenticatedState\": \"ambiguous\",\n" +
+			"            \"primary\": true\n" +
+			"          }\n" +
+			"        ],\n" +
+			"        \"USERID\": [\n" +
+			"          {\n" +
+			"            \"id\":someUserID,\n" +
```

Comment:
this should have been the string`"someUserID"` 🤔

Reply:
That isn't what it was originally, `\"id\":" + "someUserID" + ",\n"`. If it was supposed to quoted, then the original string should be `\"id\":" + "\"someUserID\"" + ",\n"`.